hwdb.d/60-evdev.hwdb

```diff
@@ -38,6 +38,15 @@
 
 # Sort by brand, model
 
+#########################################
+# ACECAD
+#########################################
+
+# Acecad Flair / Pentagram Quadpen
+evdev:input:b0003v0460p0004*
+ EVDEV_ABS_00=::20
+ EVDEV_ABS_01=::20
```

Comment:
So this should be `::40` IIUC.

Reply:
correct, apologies, should've said so explicitly

Reply:
I confirm these settings work even better ("snappier" is the best word I can think of to describe it). Commit pushed.